### PR TITLE
b2b-mp - full re-run with updated test data + updates

### PR DESCRIPTION
**Pull request by Katerynaa-Lavryk**
## PR Description

b2b-mp - full re-run with updated test data + updates

### CODE REVIEW

File: tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/negative.robot [hunk 1]
```diff
@@ -52,13 +52,12 @@ Get_availability_notifications_without_access_token
 
 #POST requests
 Subscribe_to_availability_notifications_with_invalid_sku
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/negative.robot:54]: please rename - the test is not about invalid sku, it is about on-existent sku

File: tests/api/mp_b2b/glue/checkout_endpoints/checkout/negative.robot [hunk 2]
```diff
@@ -80,8 +80,7 @@ Create_order_without_type
     [Teardown]    Run Keywords    I send a DELETE request:    /carts/${cart_id}
     ...  AND    Response status code should be:    204
 
-Create_order_with_invalid_email_&_salutation
-# Created the bug for the 'salutation' validation https://spryker.atlassian.net/browse/CC-16504
+Create_order_with_invalid_email
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/checkout_endpoints/checkout/negative.robot:83]: I see you changed the test to purely email checking. where is the test for salutation checking?

File: tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot [hunk 3]
```diff
@@ -80,45 +80,48 @@ Adding_voucher_with_invalid_access_token
...
+Deleting_voucher_without_access_token   
+# bug CC-16735
+    [Setup]    Run Keywords    I get access token for the customer:    ${yves_user_email}
+    ...    AND    I set Headers:    Authorization=${token}
+    ...    AND    I send a POST request:    /carts    {"data": {"type": "carts","attributes": {"priceMode": "${mode.gross}","currency": "${currency.eur.code}","store": "${store.de}","name": "${test_cart_name}-${random}"}}}
+    ...    AND    Save value to a variable:    [data][id]    cart_id
+    ...    AND    Response status code should be:    201
+    ...    AND    I send a POST request:    /carts/${cart_id}/items    {"data": {"type": "items","attributes": {"sku": "419901","quantity": 1}}}
+    ...    AND    Response status code should be:    201
+    ...    AND    I send a POST request:    /carts/${cart_id}/vouchers    {"data": {"type": "vouchers","attributes": {"code": "${discount.id_3.voucher_code}"}}}
```
Comment (OKrivtsova) [inline tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot:92]: please get vouchers from DB initially, not from a pre-set variable

File: tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/negative.robot (hunk 1 above)
Comment (Katerynaa-Lavryk) [reply tests/api/mp_b2b/glue/availability_endpoints/availability_notifications/negative.robot:54]: fixed

File: tests/api/mp_b2b/glue/checkout_endpoints/checkout/negative.robot (hunk 2 above)
Comment (Katerynaa-Lavryk) [reply tests/api/mp_b2b/glue/checkout_endpoints/checkout/negative.robot:83]: According to the comments in https://spryker.atlassian.net/browse/CC-16504 - there is no validation for salutation for signed in user, so I just removed this check

File: tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot (hunk 3 above)
Comment (Katerynaa-Lavryk) [reply tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot:92]: Added step to get voucher code from DB

Comment (AntonShopin) [reply tests/api/mp_b2b/glue/discount_endpoints/vouchers/negative.robot]: Please extend the steps when you get the voucher code from the DB.
First - get the voucher_pool from the DB using discount ID
Then - get the code using this voucher_pool
Also, please revome voucher codes from the env file

**Review by AntonShopin — DISMISSED** [review]

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**